airbyte-webapp/src/components/base/Button/types.tsx

```diff
@@ -1,10 +1,16 @@
+import React from "react";
+
+type ButtonSize = "xs" | "sm" | "lg";
+type ButtonVariant = "primary" | "secondary" | "danger" | "light" | "noStroke";
```

Comment:
I understand that the name in the figma is `no_stroke` but maybe we can give it a better name. How about "clear" ?